SingularityService/src/main/java/com/hubspot/singularity/resources/TaskResource.java

```diff
@@ -171,7 +171,9 @@ public TaskResource(TaskRequestManager taskRequestManager, TaskManager taskManag
   public List<SingularityPendingTaskId> getScheduledTaskIds(
       @Parameter(hidden = true) @Auth SingularityUser user,
       @Parameter(description = "Use the cached version of this data to limit expensive api calls") @QueryParam("useWebCache") Boolean useWebCache) {
-    return authorizationHelper.filterByAuthorizedRequests(user, taskManager.getPendingTaskIds(useWebCache(useWebCache)), SingularityTransformHelpers.PENDING_TASK_ID_TO_REQUEST_ID, SingularityAuthorizationScope.READ);
+    List<SingularityPendingTaskId> taskIds = authorizationHelper.filterByAuthorizedRequests(user, taskManager.getPendingTaskIds(useWebCache(useWebCache)), SingularityTransformHelpers.PENDING_TASK_ID_TO_REQUEST_ID, SingularityAuthorizationScope.READ);
+    Collections.sort(taskIds);
```

Comment:
is `taskIds` guaranteed to be mutable? It might be safer to return a sorted copy. 🤷‍♂